src/fragments/gen2/quickstart/build-a-backend.mdx

```diff
@@ -4,11 +4,11 @@ Next, we will add data and auth capabilities to the app.  In Amplify (Gen 2), th
 
 ### Add data
 
-`create-amplify` provides the scaffolding of a `/amplify/data/resource.ts` which is ready to deploy.
+`create-amplify` provides the scaffolding of a `/amplify/data/resource.ts` file, which is ready to deploy.
```

Comment:
as this fragment comes together it is a bit back and forth on the preceding slash when referencing files 
![image](https://github.com/aws-amplify/docs/assets/5033303/74dba362-be0c-4f77-b0cc-fce70b49ef25)
![image](https://github.com/aws-amplify/docs/assets/5033303/dee648f8-64f0-4271-a846-7f0712364b75)

I believe this was added to signify the project root, but this can be confusing to those newer to the space as this is typically aliased to `~` or `@` (depending on the framework) and these references are off the filesystem root. Can we remove the preceding slash from the file references?